sktime/decomposition/vamp.py

```diff
@@ -60,7 +59,7 @@ def __init__(self, cov: CovarianceModel, dim=None, epsilon=1e-6, scaling=None, r
             Singular value cutoff parameter, see :attr:`VAMP.epsilon`.
         scaling : str or None, optional, default=None
             Scaling parameter, see :attr:`VAMP.scaling`.
-        right : bool, optional, default=True
+        right : bool, optional, default=False
```

Comment:
so does changing this default to false mean the left eigenvectors are used for projection? or...?

Reply:
This is usually set from the estimator, it is just so that the defaults from the estimator match the defaults from the model. defaulting `right=False` means that not the right but the left eigenvectors are used, indeed

Reply:
might be nice to write explicitly since making it true/false w.r.t. one set is a bit hard to grok

Reply:
I have added a docstring. Normally I would say that this should not be a parameter at all but rather be specified whenever transforming data. However to obey the sklearn interface, it must be a state variable.

Reply:
thank you!